### PR TITLE
[All addons] Review all readme.md files for V3 changes.

### PR DESCRIPTION
**Pull request by Skinah**
A work in progress of going through all the readme's to remove PaperUI/Habmin and other needed changes for V3.

Completed all bindings that start with the letter M thru to Z.
I skipped Yamaha Receiver Binding for someone that knows the binding to do.

Signed-off-by: Matthew Skinner <matt@pcmus.com>

### CODE REVIEW

**Comment by Skinah** [comment]
Completed looking over bindings starting with the letters M to R (inclusive).

**Comment by lolodomo** [comment]
LGTM for thé Sonos binding.

**Comment by Confectrician** [comment]
Looks good for the general edits so far. 👍 
COn't say much about the detailled content changes though.

**Review by mherwege — APPROVED** [review]
LGTM for nikohomecontrol

**Comment by andrewfg** [comment]
[siemensrds] LGTM

**Comment by JueBag** [comment]
[Tankerkoenig]
LGTM

**Review by fwolter — CHANGES_REQUESTED** [review]
Thank you very much, also for proofreading!

File: bundles/org.openhab.binding.siemensrds/README.md [hunk 1]
```diff
@@ -75,7 +75,6 @@ Bridge siemensrds:climatixic:mybridgename "Climatix IC Account" [ userEmail="ema
 ```
 
 To manually configure an RDS Smart Thermostat Thing requires knowledge of the "Plant Id" which is a unique code used to identify a specific thermostat device in the Siemens Climatix IC cloud server account.
-The Paper UI automatic Discovery service (see above) discovers the "Plant Id" codes during the discovery process.
```
[the remark this answers is not in the document]
Comment (Skinah) [reply bundles/org.openhab.binding.siemensrds/README.md:78]: This was the second time the same thing had been repeated so it made sense to remove the second time.

File: bundles/org.openhab.binding.tankerkoenig/README.md [hunk 2]
```diff
@@ -123,28 +123,10 @@ If you receive the error because you are running an old Linux installation which
...
->> sudo dpkg -l | grep java
->> ii  oracle-java8-jdk                    8u65                             armhf        Java™ Platform, Standard Edition 8 Development Kit
-```
-
-Find the ca-store of your JDK
-
-```java
->> sudo dpkg -L oracle-java8-jdk | grep cacerts
->> /usr/lib/jvm/jdk-8-oracle-arm32-vfp-hflt/jre/lib/security/cacerts
-```
```
[the remark this answers is not in the document]
Comment (Skinah) [reply bundles/org.openhab.binding.tankerkoenig/README.md:146]: Yes it was intentional as its all on Java 8. Some of the commands also did not help on my linux system. It was mostly the last lines that I left in that was the most useful, and I figured most people would visit the forum for help on this. Upgrading Java is probably the way forward as the solution.... Happy to delete any of the commits to reverse things as I did a commit per binding.

File: bundles/org.openhab.binding.magentatv/README.md [hunk 3]
```diff
@@ -25,11 +25,11 @@ The binding provides device discovery, sending keys for the remote and also rece
 
 ## Auto Discovery
 
-UPnP will be used to discover receivers on the local network and discover the nessesary parameters.
+UPnP will be used to discover receivers on the local network and discover the necessary parameters.
 The receiver needs to be powered on to get discovered.
 
 Once the receiver is discovered it can be added from the Inbox.
-Make sure to set the U in the Thing configuration after adding the new thing, see section Thing Configuration.
+Make sure to set `udn` in the Thing configuration after adding the new thing, see section Thing Configuration.
```
[the remark this answers is not in the document]
Comment (Skinah) [reply bundles/org.openhab.binding.magentatv/README.md:32]: I cant be sure, however I would lean towards it is meant to be userId.

1. This line later on "For textual configuration at least the `ipAddress`, `udn` and `userId` parameters are required."
2. The udn looks to be parsed by the binding and the userId needs special steps to retrieve.

[the remark this answers is not in the document]
Comment (fwolter) [reply bundles/org.openhab.binding.magentatv/README.md]: I think we should go for `userId`. If it's wrong we can change it though.

[the remark this answers is not in the document]
Comment (Skinah) [reply bundles/org.openhab.binding.magentatv/README.md]: I changed it to userId. Whole sub section on how to get the userId.

**Review by fwolter — APPROVED** [review]
LGTM

File: bundles/org.openhab.binding.tankerkoenig/README.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (JueBag) [reply bundles/org.openhab.binding.tankerkoenig/README.md:146]: I haven't heard of any problems on that for recent installations, it was always a rather old installation when such problem occurred.

**Pull request merged**